src/core/pipe/pipe.sort.spec.js

```diff
@@ -0,0 +1,104 @@
+import sort from './pipe.sort';
+
+describe('sort pipe', () => {
+	const list = [{
```

Comment:
May be there would be better to set source list in more "shaked" state. Actually first test doesn't change order for asc sorting by name.